### ISSUE TITLE
Weird Depthmap on Custom Dataset

### ISSUE THREAD

**Issue by Neel1012** [TRI-ML/packnet-sfm#253]
Hi,

I trained the SelfSup model from scratch on my custom dataset with almost 13000 images with the same hyperparameters mentioned in the paper. The only change I made was changing the stride from 1 to 5 as my dataset does not have enough movement between two consecutive frames. I only take the 5th previous image and the 5th after image w.r.t. the current image and skip all the images in between them, so it works as if I am doing the training only with 3 images predicting only 2 poses.  After 20 epochs, my training loss decreased from 0.13 to 0.11. I did an inference on the checkpoint of the 20th epoch and this is how my result looks like:

![Image](https://github.com/user-attachments/assets/ab612b6a-2e52-4b14-b653-fe117760b83c)

I used the same model to infer on Kitti dataset and the result looks like this:

![Image](https://github.com/user-attachments/assets/fb792c67-340d-4942-9ee0-fd804c098f09)

I also trained the model with a stride of one with the same training setup and my inference looks like this:

![Image](https://github.com/user-attachments/assets/9361af9f-5a95-44c4-b857-78b5d649a142)

The depth map looks very sharp in the first case and I don't see anything in the second one. Is it possible to get this good depthmap or there is something wrong with the way I train the model?

Thank you
